flow-nodes/api-builder-plugin-format-date/test/test.js

```diff
@@ -1,26 +1,34 @@
 const { expect } = require('chai');
-const { MockRuntime } = require('@axway/api-builder-sdk');
-
+const { MockRuntime } = require('@axway/api-builder-test-utils');
 const getPlugin = require('../src');
 const actions = require('../src/actions');
 
 describe('flow-node format-date', () => {
-	let runtime;
-	before(async () => runtime = new MockRuntime(await getPlugin()));
+	let plugin;
+	let flowNode;
+	before(async () => {
+		plugin = await MockRuntime.loadPlugin(getPlugin);
+		flowNode = plugin.getFlowNode('format-date');
+	});
 
 	describe('#constructor', () => {
 		it('should define flow-nodes', () => {
 			expect(actions).to.be.an('object');
 			expect(actions.formatDate).to.be.a('function');
-			expect(runtime).to.exist;
-			const flownode = runtime.getFlowNode('format-date');
-			expect(flownode).to.be.a('object');
+			expect(plugin).to.be.a('object');
+			expect(plugin.getFlowNodeIds()).to.deep.equal([
+				'format-date'
+			]);
+			expect(flowNode).to.be.a('object');
 
 			// Ensure the flow-node matches the spec
-			expect(flownode.name).to.equal('Format Date');
-			expect(flownode.description).to.equal('Example flow-node that formats Date. Uses the popular Moment.js library.');
-			expect(flownode.icon).to.be.a('string');
-			expect(flownode.methods).to.deep.equal({
+			expect(flowNode.name).to.equal('Format Date');
+			expect(flowNode.description).to.equal('Example flow-node that formats Date. Uses the popular Moment.js library.');
```

Comment:
Too long, please wrap
```suggestion
			expect(flowNode.description)
				.to.equal('Example flow-node that formats Date. Uses the popular Moment.js library.');
```

Reply:
Updated.